src/sx/src/transformStyleName.js

```diff
@@ -3,5 +3,10 @@
 import * as changeCase from 'change-case';
 
 export default function transformStyleName(styleName: string): string {
+  if (styleName.startsWith('--')) {
+    // CSS variable
+    return styleName;
```

Comment:
It already works with CSS variables, what makes this change necessary?

Reply:
See the `src/sx/src/__tests__/transformStyleName.test.js`. It was transforming the variable name.

Reply:
Aha, I see, this is for defining the variable, not using it 👍